### PR TITLE
Wire up all tx page

### LINKED ISSUE #743: Wire up all transactions page

### PR DESCRIPTION
**Pull request by prin-r**
Fix #743

### CODE REVIEW

File: scan/src/rpc/TxHook.re [hunk 1]
```diff
@@ -308,7 +308,8 @@ module Tx = {
     timestamp: MomentRe.Moment.t,
     gasWanted: int,
     gasUsed: int,
-    fee: Coin.t,
+    fee: float,
```
Comment (taobun) [inline scan/src/rpc/TxHook.re]: Why we change type of fee from coin to float?

File: scan/src/components/msgs/Msg.re [hunk 2]
```diff
@@ -173,22 +200,33 @@ let make = (~msg: TxHook.Msg.t, ~width: int) => {
...
-          color=Colors.mediumGray
-          weight=Text.Medium
-          nowrap=true
-          block=true
-          ellipsis=true
-        />
-      </div>
+      {success
+         ? <>
+             <TypeID.Request id={ID.Request.ID(id)} />
```
Comment (taobun) [inline scan/src/components/msgs/Msg.re:205]: Why request id equal to 0

**Review by taobun — CHANGES_REQUESTED** [review]

**Comment by taobun** [comment]
@prin-r  Cannot watch https://d3n-scan-pr-792.onrender.com/

File: scan/src/rpc/TxHook.re (hunk 1 above)
Comment (prin-r) [reply scan/src/rpc/TxHook.re]: done

File: scan/src/components/msgs/Msg.re (hunk 2 above)
Comment (prin-r) [reply scan/src/components/msgs/Msg.re:205]: done

File: scan/src/rpc/TxHook.re [hunk 3]
```diff
@@ -72,7 +78,7 @@ module Msg = {
 
     let decode = json =>
       JsonUtils.Decode.{
-        id: 952, // TODO, wire up
+        id: json |> field("dataSourceID", intstr),
```
Comment (taobun) [inline scan/src/rpc/TxHook.re]: Add TODO: for event

**Review by taobun — CHANGES_REQUESTED** [review]

File: scan/src/rpc/TxHook.re (hunk 3 above)
Comment (prin-r) [reply scan/src/rpc/TxHook.re]: done

**Review by taobun — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]

**Pull request merged**